jiant/tasks/tasks.py

```diff
@@ -3731,3 +3732,121 @@ def get_metrics(self, reset=False):
         """Get metrics specific to the task"""
         acc = self.scorer1.get_metric(reset)
         return {"accuracy": acc}
+
+
+@register_task("wikipedia_corpus_sop", rel_path="wikipedia_corpus_small")
+class SentenceOrderTask(PairClassificationTask):
+    """ Task class for Sentence Order Prediction (SOP) with wikipedia data.
+        See the ALBERT paper for details on SOP: https://arxiv.org/abs/1909.11942.
+        We are currently using an unpreprocessed version of the Wikipedia corpus
```

Comment:
Say a tiny bit more about the data source here, even if the full docs live elsewhere—there is no one 'Wikipedia corpus'.